pmt/test/qa_pmt.cpp

```diff
@@ -99,4 +101,63 @@ TEST(Pmt, VectorWrites)
 
         EXPECT_EQ(int_pmt_vec->value(), int_vec_val_modified);
     }
-}
\ No newline at end of file
+}
+
+TEST(Pmt, VectorWrapper) {
+    pmt_vector_wrapper<uint32_t> x(10);
```

Comment:
I'm guessing `pmt_vector_wrapper` would become `pmt_vector` and `pmt_vector` would become something like `pmt_vector_base` ?

Reply:
Yes